package.json

```diff
@@ -25,7 +25,7 @@
   "devDependencies": {
     "bootstrap": "^4.1.1",
     "browserify": "latest",
-    "grunt": "^1.0.4",
+    "grunt": "^1.0.0",
```

Comment:
Why are you doing this? I think ^1.0.4 and ^1.0.0 will ultimately download same package.